docs/sql-ref-functions-udf-aggregate.md

```diff
@@ -113,26 +102,26 @@ OPTIONS (
 );
 
 SELECT * FROM employees;
--- +-------+------+
--- |   name|salary|
--- +-------+------+
--- |Michael|  3000|
--- |   Andy|  4500|
--- | Justin|  3500|
--- |  Berta|  4000|
--- +-------+------+
++-------+------+
+|   name|salary|
++-------+------+
+|Michael|  3000|
+|   Andy|  4500|
+| Justin|  3500|
+|  Berta|  4000|
++-------+------+
 
 SELECT myAverage(salary) as average_salary FROM employees;
--- +--------------+
--- |average_salary|
--- +--------------+
--- |        3750.0|
--- +--------------+
-{% endhighlight %}
++--------------+
+|average_salary|
++--------------+
+|        3750.0|
++--------------+
+```
 </div>
```

Comment:
We cannot avoid this tag, too?

Reply:
I will take a look at this.

Reply:
This is for examples ```<div class="codetabs">```. I prefer to keep this since we use this format for all the examples.